app/controllers/container_topology_controller.rb

```diff
@@ -18,7 +18,7 @@ def index
     redirect_to :action => 'show'
   end
 
-  def data
+  def topology_data
```

Comment:
> why does this method require renaming?

Just to make topology api sort of uniform until the topology tab gets removed.
in any controller, the topology method shouldnt just be named data